src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/codegen_gelu.cpp

```diff
@@ -13,15 +13,42 @@ namespace ov {
 namespace test {
 namespace snippets {
 namespace {
+std::vector<InputShape> inShapes0{
+    {{}, {{1, 384, 4096}}},
+    {{}, {{1, 1, 4096}}},
+};
+std::vector<InputShape> inShapes1{
+    {{}, {{1, 384, 4096}}},
+    {{}, {{1, 384, 1}}},
+};
 
-    INSTANTIATE_TEST_SUITE_P(NoReshape, CodegenGelu,
-            ::testing::Combine(
-            ::testing::Values(ov::element::f32),
-            ::testing::Values(ov::Shape {1, 384, 4096}),
-            ::testing::Values(true, false),
-            ::testing::Values(CommonTestUtils::DEVICE_CPU)),
-            CodegenGelu::getTestCaseName);
-}  // namespace
+INSTANTIATE_TEST_SUITE_P(NoReshapeAndReshape, CodegenGelu,
+        ::testing::Combine(
+        ::testing::Values(ov::element::f32),
+        ::testing::ValuesIn(inShapes0),
+        ::testing::ValuesIn(inShapes1),
+        ::testing::Values(true, false),
+        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+        CodegenGelu::getTestCaseName);
+
+// DS
+std::vector<InputShape> inShapesDynamic0{
+    {{-1, -1, -1}, {{1, 384, 4096}, {1, 384, 1}, {1, 384, 4096}}},
```

Comment:
I'd suggest to reduce Shapes to avoid big execution for simple test (the main idea - to have innermost Loop with several iterations, so you can set shape smth like `1x12x128`). What do you think?

Reply:
updated, also for static shape case as seems like the big size is not intentional and not a must have.